app/lib/features/attachments/widgets/attachment_section.dart

```diff
@@ -122,146 +80,76 @@ class FoundAttachmentSectionWidget extends ConsumerWidget {
     );
   }
 
-  Widget _buildAttachmentItem(
+  Widget attachmentData(
+    List<Attachment> list,
     BuildContext context,
-    Attachment item,
-    bool canEdit,
+    WidgetRef ref,
   ) {
-    final eventId = item.attachmentIdStr();
-    final roomId = item.roomIdStr();
-    return Stack(
-      children: [
-        AttachmentItem(
-          key: Key('$eventId-attachment'),
-          attachment: item,
-        ),
-        Positioned(
-          top: -12,
-          right: -12,
-          child: Visibility(
-            visible: canEdit,
-            child: IconButton(
-              key: AttachmentSectionWidget.redactBtnKey,
-              onPressed: () => openRedactContentDialog(
-                context,
-                eventId: eventId,
-                roomId: roomId,
-                title: L10n.of(context).deleteAttachment,
-                description:
-                    L10n.of(context).areYouSureYouWantToRemoveAttachmentFromPin,
-                isSpace: true,
-                removeBtnKey: AttachmentSectionWidget.confirmRedactKey,
-              ),
-              icon: const Icon(
-                Atlas.minus_circle_thin,
-                size: 14,
-              ),
-            ),
+    bool canEdit = attachmentManager.canEditAttachments();
+    return Padding(
+      padding: const EdgeInsets.all(12),
+      child: Column(
+        crossAxisAlignment: CrossAxisAlignment.start,
+        children: <Widget>[
+          attachmentHeader(context, ref),
+          if (list.isEmpty) ...[
+            const SizedBox(height: 10),
+            Text(L10n.of(context).attachmentEmptyStateTitle),
+          ],
+          Wrap(
+            spacing: 5.0,
+            runSpacing: 10.0,
+            children: <Widget>[
+              if (list.isNotEmpty)
+                for (var item in list)
```

Comment:
```suggestion
                for (var item in list)
```

unnecessary.

if at all it makes sense to put the entire `Wrap` behind the if ...